src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-proguard-or-dexguard-android-apps.mdx

```diff
@@ -29,6 +29,16 @@ To add support for [ProGuard](http://proguard.sourceforge.net/manual/usage.html)
    -keep class com.newrelic.** { *; }
    -dontwarn com.newrelic.**
    -keepattributes Exceptions, Signature, InnerClasses, LineNumberTable, SourceFile, EnclosingMethod
+
+  ##
+  ## NewRelic Gradle plugin 7.x requires the following additions:
```

Comment:
Please change "requires" to "may require". Not all apps will encounter this issue.